Documentation/proposals/202304-silences.md

```diff
@@ -0,0 +1,136 @@
+# Silence CRD
+
+* Owners:
+  * [mcbenjemaa](https://github.com/mcbenjemaa)
+* Related Tickets:
+  * [#5452](https://github.com/prometheus-operator/prometheus-operator/issues/5452)
+  * [#2398](https://github.com/prometheus-operator/prometheus-operator/issues/2398)
+* Other docs:
+  * n/a
+
+This document describes the creation of `Silence` Custom Resource Definition that defines Alertmanager silences.
+
+# Why
+
+Prometheus-operator does not provide a way to automate the management of Alertmanager silences and there is no current standard/best practice way to do that.
+Moreover, setting an automation to manage silences has been reported as cumbersome by multiple users.
+or an additional operator that does the job e.g. [silence-operator](https://github.com/giantswarm/silence-operator).
```

Comment:
this part doesn't read properly with the previous line.

Reply:
let me rephrase it.